fe/fe-core/src/main/java/com/starrocks/transaction/TransactionChecker.java

```diff
@@ -0,0 +1,79 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.transaction;
+
+import com.starrocks.catalog.Database;
+import com.starrocks.catalog.LocalTablet;
+import com.starrocks.catalog.MaterializedIndex;
+import com.starrocks.catalog.OlapTable;
+import com.starrocks.catalog.Partition;
+import com.starrocks.catalog.Tablet;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionChecker {
```

Comment:
please give a brief introduction to this class

Reply:
ok